### PR TITLE
[WIP] Raw docs

### PR DESCRIPTION
**Pull request by helfi92**
Resolves https://github.com/taskcluster/taskcluster-web/issues/318.
Refers to https://github.com/taskcluster/taskcluster-web/issues/54.

### CODE REVIEW

File: src/scripts/generateToc.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
 const fs = require('fs');
```
Comment (djmitche) [inline src/scripts/generateToc.js:1]: Will this script end up being part of `yarn generate`?

Note that `yarn generate` now moves much of the docs data to `generated/` -- that may be a "head start" on getting it imported into the docs site.

**Review by djmitche — COMMENTED** [review]
This is a *lot* of code, and it's hard to understand the overall approach from this level of detail.  Can you write down the design you're using?

File: src/scripts/generateToc.js (hunk 1 above)
Comment (helfi92) [reply src/scripts/generateToc.js:1]: Yes, I believe it should.

> Note that `yarn generate` now moves much of the docs data to `generated/`

That's awesome. Thanks :)

**Comment by helfi92** [comment]
> This is a _lot_ of code, and it's hard to understand the overall approach from this level of detail. Can you write down the design you're using?

Yeah, I figured. I'll write something up soon.

**Comment by djmitche** [comment]
Feel free to change the generation process to put that data somewhere else than generated/ if it's more convenient for you.

**Pull request closed without merging**